docs/advanced-topics/advanced-methods.md

```diff
@@ -0,0 +1,100 @@
+(advanced-methods)=
+# {octicon}`mortar-board` Advanced Methods
+Many of the models supported by finetuner make use of similar methods throughout finetuning, such as the methods used for *calculating loss*, *sampling* and *pooling* . Finetuner offers alternatives to each of these methods, which can improve the performance of your finetuning run in some cases.
```

Comment:
This sounds a bit generic to my. I would suggest to split this section into two sections "Loss functions" and "Pooling Layers"